docs/contributing.md

```diff
@@ -3,46 +3,93 @@
 Thank you for being interested in contributing to the `sphinx-book-theme`! You
 are awesome ✨.
 
-This page contains developer documentation to help you get started.
+This project follows the Executable Books Project [contribution guidelines](https://executablebooks.org/en/latest/contributing.html).
+It contains information about our conventions around coding style, pull request workflow, commit messages and more.
 
-```{toctree}
-:hidden:
-contributing-ebp
+This page contains information to help you get started with development on this
+project.
+
+## Development
+
+### Set-up
+
+Get the source code of this project using git:
+
+```bash
+git clone https://github.com/executablebooks/sphinx-book-theme
+cd sphinx-book-theme
 ```
 
-## The EBP Contributing Guide
+To work on this project, you need Python 3.6 or newer. Most of this project's
+development workflow commands use `nox`, which can be installed with pip:
```

Comment:
can you briefly explain what nox is, and link to the nox documentation? Assume a reader has never heard of nox and is only marginally familiar with HTML/CSS workflows

Reply:
Will do in a follow up PR!

Reply:
Filed #178 for this.